Update vars.md (#41)
compono/idibu-api: posting-api/vars.md

<h1>
	Configuration field variables</h1>

> <p>Each of this variables are to be sent (if needed) in the configuration block of the XML request, that is enclosed in the config tag.</p>

Reviewer: ```suggestion
<p>Each of these variables is to be sent (if needed) in the configuration block of the XML request, which is enclosed in the config tag.</p>
```
